.gitpod.Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM gitpod/workspace-python
 RUN pyenv install 3.7.12
-RUN pipenv sync --dev
\ No newline at end of file
+RUN pipenv sync --dev
```

Comment:
This here doesn't work currently due to the repo not being pulled in until the workspace is loaded. I'm going to have this in the init phase instead.